portfolio/src/main/webapp/lucygallery.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Lucy's Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body bgcolor = "#66819c">
+    <div id="navbar">
+	<ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="lucygallery.html">Lucy Gallery</a></li>
+        <li><a href="#projects">Projects</a></li>
+        <li><a href="#suprise">Suprise</a></li>
+      </ul>
+    </div>
+    <div id="content">
+      <h1>Lucy's Gallery</h1>
+        <p>Here are some images of my cat, Lucy! She's wonderful and very fiesty :3</p>
+    </div>
+    <div class="row" >
+      <div class="col">
+        <img src="images/lucy1.jpg" width="75%"/>
```

Comment:
Two things about these width attributes:

1. They should be done in CSS.
2. Percentage is one of the least desirable ways of sizing things because it can have incredibly unexpected behavior. Can you specify a width in pixels instead?